src/components/ButtonMenu/readme.md

```diff
@@ -454,3 +454,254 @@ class ButtonMenuExample extends React.Component {
+    border: 1px solid ${props => props.palette.brand.main};
+    background-color: ${props => props.palette.brand.main};
+    width: 98%;
+    padding: 0;
+    margin: 0;
+`;
+
+const SegmentContainer = styled.div`
+    width: 100%;
+    height: 20px;
+`;
+
+const Segment = props => {
+    return (
+        <SegmentContainer className="rainbow-flex rainbow-align_center">
+            <Point />
+            <Line />  
+            <Point />  
+        </SegmentContainer>
+    );
+}
+
+function getdDateDiference ( dateIni, dateEnd ) {
+    var dateIni_ms = dateIni.getTime();
+    var dateEnd_ms = dateEnd.getTime();
+    var difference_ms = dateEnd_ms - dateIni_ms;
+    difference_ms = difference_ms/1000/60; 
+    const minutes = Math.floor(difference_ms % 60);
+    difference_ms = difference_ms/60; 
+    const hours = Math.floor(difference_ms);
+    
+    return (hours<10?'0':'')+hours+'hr ' + minutes + 'min';
+}
+
+function getFormatedTime(date) {
+    const rawHours  = date.getHours();
+    let h = rawHours > 12 ? rawHours%12 : (rawHours===0 ? 12 : rawHours);
+    h = h<10?'0'+h:h;
+    const m = date.getMinutes();
+    const format = rawHours>=12?'PM':'AM';
+    return `${h}:${m} ${format}`;
+}
+
+function getMonth(date) {
+    const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    return months[date.getMonth()];
+}
+
+function FlightCard( props ) {
+    const { 
+        departureDate,
+        arrivalDate,
+        origin,
+        originAbbreviation,
+        destination,
+        destinationAbbreviation,
+        airline,
+        flightNumber,
+        price
+    } = props;
+
+    return (
+        <div className="rainbow-m-top_small">
+            <CardSubtitle>
+                Departure flight: { origin } - { destination } • {getMonth(departureDate)} {departureDate.getDate()}
+            </CardSubtitle>
+            <Card>
+                <div className="rainbow-flex rainbow-flex_wrap">
+                    <Row>
+                        <Column
+                            flex="1"
+                            label={originAbbreviation}
+                            description={getFormatedTime(departureDate)}
+                            noBorder
+                        />
+                        <Column
+                            flex="1.3"
+                            noBorder
+                            description={getdDateDiference(departureDate, arrivalDate)}
+                        >
+                            <Segment />
+                        </Column>
+                        <Column
+                            flex="1"
+                            label={destinationAbbreviation}
+                            description={getFormatedTime(arrivalDate)}
+                            noBorder
+                        />
+                    </Row>
+                    <Column label={airline} description="Airline" specialColor />
```

Comment:
Since the meaning of specialColor is to use the brand color let's better use isBrand, hasBrand or hasBrandLabel (this is more specific about what will be brand). For convention we always prefix all boolean values with is or has to indicate true or false